nurseconnect/templates/core/header.html

```diff
@@ -13,7 +13,7 @@
             <nav class="NavBar NavBar--siteHead" role="menubar">
                 <ul class="NavBar-navGroup" role="menu">
                     <li class="NavBar-navItem{% if active == "menu" %} NavBar-navItem--active{% endif %}"
-                        role="presentation">
+                        role="presentation" data-enhanced-nav>
```

Comment:
Ah, because it hooks to a menu? Since the others are menuitem. Makes sense I guess.